### PR TITLE
Fjc arrays

### PR DESCRIPTION
**Pull request by andriioleshko**

### CODE REVIEW

File: src/main/java/arrays/Main.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+            newArray[i] = oldArray[i];
+        }
+        return newArray;
+    }
+
+    static int sumOfArrayElements(int[] array, boolean onlyPositives) {
+        int result = 0;
+        for (int el : array) {
+            if (onlyPositives && el <= 0) {
+                continue;
```
Comment (Maks-Chornyi) [inline src/main/java/arrays/Main.java]: please never use break/continue/goto (in c# for instance) in your coding.

File: src/main/java/arrays/Main.java [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+            }
+            result += el;
+        }
+        return result;
+    }
+
+    static void removeUnnecessaryElements(ArrayList<Integer> arrayList) {
+        arrayList.removeIf(el -> el % 3 == 0);
```
Comment (Maks-Chornyi) [inline src/main/java/arrays/Main.java]: imagine that tomorrow I will ask you to have similar logic, but to remove elements which has no 
Returns the division remainder not by three, but by two. In addition, I will ask same in one week after but for number 7. 
Would you create additional methods?

hint: provide method which can work not just with numbers which has no division remainder for 3, but for different values.

**Review by Maks-Chornyi — COMMENTED** [review]
left you comments, check them out.

File: src/main/java/arrays/Main.java (hunk 2 above)
Comment (andriioleshko) [reply src/main/java/arrays/Main.java]: I could use parameters, but it was not mentioned, that value could be changed. In other way I would add this patameter to the method while creating it.

File: src/main/java/arrays/Main.java (hunk 1 above)
Comment (andriioleshko) [reply src/main/java/arrays/Main.java]: Why not? Is there a recommendation not to use this (or other) operators?

Comment (andriioleshko) [reply src/main/java/arrays/Main.java]: @Maks-Chornyi used another construction inside of the loop

File: src/main/java/arrays/Main.java (hunk 2 above)
Comment (andriioleshko) [reply src/main/java/arrays/Main.java]: @Maks-Chornyi created a new variable, tha use in the method where divide numbers and in the text to print out

File: src/main/java/arrays/Main.java [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        Integer[] newArray = new Integer[oldArray.length];
+        for (int i = 0; i < oldArray.length; i++) {
+            newArray[i] = oldArray[i];
+        }
+        return newArray;
+    }
+
+    static int sumOfArrayElements(int[] array, boolean onlyPositives) {
+        int result = 0;
+        for (int el : array)
```
Comment (Maks-Chornyi) [inline src/main/java/arrays/Main.java]: you can try to read java convention once you have time. It says you should never omit curly braces in statements (if/for/while so on). this is common good practice.

**Review by Maks-Chornyi — COMMENTED** [review]
fix this little thing and we gonna deliver these changes to master

File: src/main/java/arrays/Main.java (hunk 3 above)
Comment (andriioleshko) [reply src/main/java/arrays/Main.java]: @Maks-Chornyi thnx for noticing

**Review by Maks-Chornyi — APPROVED** [review]

**Pull request merged**